RemoteCommands/TealiumFirebase/plugin.xml

```diff
@@ -21,13 +21,14 @@
 
     <!-- ios -->
     <platform name="ios">
+        <preference name="deployment-target" value="15.0" />
         <plugins-plist key="TealiumFirebase" string="TealiumFirebase" />
         <config-file target="config.xml" parent="/*">
             <plugin name="TealiumFirebase" value="TealiumFirebase"/>
             <feature name="TealiumFirebase">
                 <param name="ios-package" value="TealiumFirebase" />
             </feature>
-            <preference name="SwiftVersion" value="5" />
+            <preference name="SwiftVersion" value="6" />
```

Comment:
Did you have issues with `v5` - as the CocoaPods package specifies [5.0](https://github.com/Tealium/tealium-ios-firebase-remote-command/blob/main/TealiumFirebase.podspec#L22C5-L22C28)

Perhaps @Enricoza could comment on the knock-on effect of updating?